src/Microsoft.DotNet.Build.Tasks/PackageFiles/publishtest.targets

```diff
@@ -260,10 +260,9 @@
     </PropertyGroup>
 
     <!-- the project json and runner script files need to be included in the archive -->
-    <Copy SourceFiles="$(ProjectJson);$(ProjectLockJson)" DestinationFolder="$(OutDir)" />
-    <Copy SourceFiles="$(OutputPathForScriptGenerator)" DestinationFolder="$(OutDir)" />
```

Comment:
If this is referenced in multiple locations - "$(TestPath)%(TestNugetTargetMoniker.Folder)", you may want to create an itemgroup just for this path structure to ensure that any changes are propogated to all of the right places.